README.md

```diff
@@ -1,122 +1,49 @@
 # NeMo Skills
 
-NeMo-Skills is a collection of pipelines to improve "skills" of large language models. You can use it to generate synthetic data, train/evaluate models, analyzing outputs and more!
+NeMo-Skills is a collection of pipelines to improve "skills" of large language models. You can use it to generate synthetic data, train/evaluate models, analyze model outputs and more!
```

Comment:
How exactly is NeMo-Skills supporting model output analysis?

Reply:
We used to have inspector, but not anymore I guess. Let me remove that